settings.d/host_reports.yml.example

```diff
@@ -13,4 +13,10 @@
 # Keep sent reports in spool_dir/done directory
 # - when enabled, move files from the place on
 # a regular basis (e.g. via cronjob).
-:keep_reports: false
\ No newline at end of file
+:keep_reports: false
+
+# Development settings (do not use)
+# Override hostnames of incoming reports
+#:override_hostname: report.example.com
```

Comment:
If I don't touch it as a user (not a developer) then I will encounter:
```
2021-10-08T15:36:42  [W] Error details for Disabling all modules in the group ['host_reports'] due to a failure in one of them: Parameter 'override_hostname' is expected to have a non-empty 
value: <Proxy::Error::ConfigurationError>: Parameter 'override_hostname' is expected to have a non-empty value
/home/vagrant/smart-proxy/lib/proxy/plugin_validators.rb:41:in `validate!'
/home/vagrant/smart-proxy/lib/proxy/plugin_validators.rb:21:in `evaluate_predicate_and_validate!'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:351:in `block in execute_validators'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:348:in `each'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:348:in `inject'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:348:in `execute_validators'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:341:in `validate_settings'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:288:in `load_settings'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:263:in `load_plugin_settings'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:80:in `load_plugin_settings'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:167:in `each'
/home/vagrant/smart-proxy/lib/proxy/plugin_initializer.rb:167:in `initialize_plugins'
/home/vagrant/smart-proxy/lib/launcher.rb:171:in `launch'
/home/vagrant/smart-proxy/bin/smart-proxy:6:in `<top (required)>'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `load'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/libexec/bundle:46:in `block in <top (required)>'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/opt/rh/rh-ruby27/root/usr/share/gems/gems/bundler-2.1.4/libexec/bundle:34:in `<top (required)>'
/opt/rh/rh-ruby27/root/usr/bin/bundle:23:in `load'
/opt/rh/rh-ruby27/root/usr/bin/bundle:23:in `<main>'
```

Reply:
Fixed.